frontend/src/components/cluster/ClusterChooser.tsx

```diff
@@ -25,7 +32,11 @@ export default function ClusterChooser({ clickHandler, cluster }: ClusterChooser
 
   return (
     <Button size="large" variant="contained" onClick={clickHandler} className={classes.button}>
-      <Trans t={t}>Cluster: {{ cluster }}</Trans>
+      <Trans t={t}>
+        <span className={classes.clusterName} title={cluster}>
+          Cluster: {{ cluster }}
+        </span>
+      </Trans>
```

Comment:
AFAIU, the Trans function should be in the span and wrapping just the text. Otherwise we'll get the tags in the translation and it's not needed because they wrap the whole phrase.
In other words, tags are fine within `<Trans>` if they're not just wrapping the full text, eg.:
`<Trans>My not bold word and my <b>bold word</b>!</Trans>`